kafka-rest/src/test/java/io/confluent/kafkarest/v2/KafkaConsumerManagerTest.java

```diff
@@ -133,28 +133,180 @@ public void testConsumerOverrides() {
         EasyMock.verify(consumerFactory);
     }
 
+    /**
+     * consumer.request.timeout.ms should not modify how long the proxy waits until returning a response.
+     * fetch.max.wait.ms should dictate that
+     */
     @Test
-    public void testConsumerNormalOps() throws InterruptedException, ExecutionException, RestConfigException {
-        config = new KafkaRestConfig(setUpProperties(), new SystemTime());
-        consumerManager = new KafkaConsumerManager(config, consumerFactory);
-        consumer = new MockConsumer<>(OffsetResetStrategy.EARLIEST);
-        // Tests create instance, read, and delete
-        final List<ConsumerRecord<byte[], byte[]>> referenceRecords
-                = Arrays.<ConsumerRecord<byte[], byte[]>>asList(
-                new BinaryConsumerRecord(topicName, "k1".getBytes(), "v1".getBytes(), 0, 0),
-                new BinaryConsumerRecord(topicName, "k2".getBytes(), "v2".getBytes(), 0, 1),
-                new BinaryConsumerRecord(topicName, "k3".getBytes(), "v3".getBytes(), 0, 2)
-        );
+    public void testConsumerRequestTimeoutDoesNotModifyProxyResponseTime() throws Exception {
+        Properties props = setUpProperties(new Properties());
+        props.setProperty(KafkaRestConfig.CONSUMER_REQUEST_TIMEOUT_MS_CONFIG, "2500");
+        setUpConsumer(props);
+
+        expectCreate();
+        EasyMock.replay(mdObserver, consumerFactory);
+        String cid = consumerManager.createConsumer(
+            groupName, new ConsumerInstanceConfig(EmbeddedFormat.BINARY));
+        consumerManager.subscribe(groupName, cid, new ConsumerSubscriptionRecord(Collections.singletonList(topicName), null));
+
+        readFromDefault(cid);
+        assertEquals(config.getInt(KafkaRestConfig.PROXY_FETCH_MAX_WAIT_MS_CONFIG),
+            Integer.parseInt(KafkaRestConfig.PROXY_FETCH_MAX_WAIT_MS_DEFAULT));
+        Thread.sleep(config.getInt(KafkaRestConfig.PROXY_FETCH_MAX_WAIT_MS_CONFIG));
```

Comment:
In the `KafkaConsumerManagerTest` we need to call `Thread.sleep()` as `KafkaConsumerReadTask` does not implement the `Future` interface like the v1 `ConsumerReadTask` does.